understanding/20/headings-and-labels.html

```diff
@@ -26,46 +26,43 @@ <h2>Intent of Headings and Labels</h2>
+            marked up/identified), while failing this Success Criterion (if those headings or labels are not
+            sufficiently clear or descriptive).
          </p>
-         
+         <p>Further, in the case of labels, this Success Criterion does not take into consideration whether or not
+            alternative methods of providing an accessible name for form controls and inputs has been
+            used - this aspect is covered separately by <a href="name-role-value">4.1.2: Name, Role and Value</a>. It is possible
+            for controls and inputs to have an appropriate accessible name (e.g. using <code>aria-label="..."</code>)
+            and therefore pass Success Criterion 4.1.2, but to still fail this Success Criterion (if the label is not
+            sufficiently clear or descriptive).
+         </p>
+         <p>While this Success Criterion does not <em>require</em> the use of labels (only ensuring that if labels
+            are present, they must be sufficiently clear or descriptive), this aspect is covered separately by
+            <a href="labels-or-instructions">3.3.2: Labels or Instructions</a>.
+         </p>         
       </div>
       
-      
    </section>
    <section id="benefits">
       <h2>Benefits of Headings and Labels</h2>
       
-      
       <ul>
          
          <li>Descriptive headings are especially helpful for users who have disabilities that make
             reading slow and for people with limited short-term memory. These people benefit when
             section titles make it possible to predict what each section contains.
          </li>
-         
-         <li>People who have difficulty using their hands or who experience pain when doing so
-            will benefit from techniques that reduce the number of keystrokes required to reach
-            the content they need.
-         </li>
-         
-         <li>
-            
-            <p>This Success Criterion helps people who use screen readers by ensuring that labels
-               and headings are meaningful when read out of context, for example, in a Table of Contents,
-               or when jumping from heading to heading within a page.
-            </p>
-            
-            <p>This Success Criterion may also help users with low vision who can see only a few
-               words at a time. 
-               
-            </p>
-            
-         </li>
-         
+
+         <li>When headings and labels are also correctly marked up and identified in accordance with <a href="info-and-relationships">1.3.1: Info and Relationships</a>, this Success Criterion helps people who use screen readers by ensuring that labels and headings are meaningful when read out of context, for example, in an automatically generated list of headings/table of contents, or when jumping from heading to heading within a page.</li>
+        
```

Comment:
<li>Form input controls with labels that clearly describe the content that is expected to be input helps users know how to successfully complete the form.</li>

Reply:
> * Form input controls with labels that clearly describe the content that is expected to be input helps users know how to successfully complete the form.

good suggestion, added